src/libcore/option.rs

```diff
@@ -762,12 +763,21 @@ impl<T> AsSlice<T> for Option<T> {
 
 #[stable]
 impl<T> Default for Option<T> {
-    #[stable]
     #[inline]
     #[stable]
     fn default() -> Option<T> { None }
 }
 
+#[stable]
+impl<T: fmt::String> fmt::String for Option<T> {
```

Comment:
Are we sure we want this? I know I've had bad output before with `println!("x is {}", x)` where `x` was unintentionally an `Option`.